tests/docker_cluster.py

```diff
@@ -67,7 +67,7 @@ def __init__(self, master_host, slave_hosts,
     def all_hosts(self):
         return self.slaves + [self.master]
 
-    def all_internal_hosts(self):
+    def all_internal_hosts(self, stopped_host=None):
```

Comment:
oh, I remember why I had it. it's because you don't know which host is the stopped host and the name is wrong.